Modules/core/Controller/CorespaceaccessController.php

```diff
@@ -48,7 +48,7 @@ public function indexAction($id_space, $letter = "A", $active = "") {
                 $letter = $_SESSION["user_last_letter"];
             }
             else{
-                $letter = "A";
+                $letter = "All";
```

Comment:
why settings to all by default?

Reply:
It was obviously UX consideration : it seems to be more user-friendly to get all users at once, at most in the case there is not a big amount of them. But, looking at the number of requests actually required when $letter is set to "All", it is way too expensive in terms of resources.
I set it back to "A".